CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We fixed an issue where there was a failure to access the url link for "eprint" for the ArXiv entry.[#10474](https://github.com/JabRef/jabref/issues/10474)
 - We fixed an issue where it was not possible to connect to a shared database once a group with entries was added or other metadata modified [#10336](https://github.com/JabRef/jabref/issues/10336)
 - We fixed an issue where middle-button paste in X not always worked [#7905](https://github.com/JabRef/jabref/issues/7905)
+- We fixed issues where in 1) preferences: external file type duplicates when change languages. [#10271](https://github.com/JabRef/jabref/issues/10271) 2) Cancel button only close EditExternalFileTypeEntryDialog without further action 3) Missed value in EditExternalFileTypeEntryDialog cannot close dialog even OK button clicked 4) Cannot save new External File Type if it extension exists already
```

Comment:
Way too long text. Just shorten it. Maybe: "We fixed issues in the external file type dialog, e.g., duplicate entries in the case of a language switch. [#10271](https://github.com/JabRef/jabref/issues/10271)"

Reply:
thanks, maybe I cover too much changes in a ticket, sorry about that.
I shorten the description and pushed a commit.